src/controllers/board.controller.ts

```diff
@@ -0,0 +1,69 @@
+import { Request, Response, NextFunction } from "express";
+import Board from "../models/Board";
+import List from "../models/List";
+import Task from "../models/Task";
+import { NotFoundError } from "../errors/NotFoundError";
+import sequelize from "../config/sequelize";
+
+export const BoardController = {
+  async getAll(req: Request, res: Response) {
+    const boards = await Board.findAll({
+      include: { model: List, include: [Task] },
+      order: [["createdAt", "ASC"]],
+    });
+    res.json(boards);
+  },
+
+  async getOne(req: Request, res: Response, next: NextFunction) {
+    const board = await Board.findByPk(req.params.id, {
+      include: {
+        model: List,
+        include: [
+          {
+            model: Task,
+            separate: true,
+            order: [["taskOrder", "ASC"]],
+          },
+        ],
+      },
+      rejectOnEmpty: new NotFoundError("Board not found"),
+    });
+    res.json(board);
+  },
+
+  async create(req: Request, res: Response) {
+    const { title } = req.body;
+    const result = await sequelize.transaction(async (transaction) => {
+      const board = await Board.create({ title }, { transaction });
+      return board;
+    });
+    res.status(201).json(result?.get({ plain: true }));
```

Comment:
You should always have a result and you don't need to do `get plain` as `.json()` will do it for you